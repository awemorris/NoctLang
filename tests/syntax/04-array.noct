func main() {
   array1 = [0, 1, 2];
   for (v in array1) {
      print(v);
   }

   print("Length = " + array1.length);

   array2 = [];
   array2[0] = 0;
   array2[1] = 1;
   array2[2] = 2;
   array2->push(3);
   for (v in array2) {
      print(v);
   }

   array2->resize(2);
   for (v in array2) {
      print(v);
   }
}
