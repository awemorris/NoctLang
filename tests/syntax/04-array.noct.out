0
1
2
Length = 3
0
1
2
3
0
1
